[2022-08-23 11:47:48,873] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-20T04:00:00+00:00 [queued]>
[2022-08-23 11:47:48,894] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-20T04:00:00+00:00 [queued]>
[2022-08-23 11:47:48,894] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:47:48,895] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-23 11:47:48,895] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 11:47:48,923] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): print_world> on 2021-10-20 04:00:00+00:00
[2022-08-23 11:47:48,988] {standard_task_runner.py:52} INFO - Started process 985 to run task
[2022-08-23 11:47:48,995] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'raizen_lab_01', 'print_world', 'scheduled__2021-10-20T04:00:00+00:00', '--job-id', '2343', '--raw', '--subdir', 'DAGS_FOLDER/raizen_lab_01.py', '--cfg-path', '/tmp/tmpyo28dvmi', '--error-file', '/tmp/tmp04m4d8yv']
[2022-08-23 11:47:48,996] {standard_task_runner.py:80} INFO - Job 2343: Subtask print_world
[2022-08-23 11:47:49,121] {task_command.py:371} INFO - Running <TaskInstance: raizen_lab_01.print_world scheduled__2021-10-20T04:00:00+00:00 [running]> on host fc645605a611
[2022-08-23 11:47:49,244] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=admin
AIRFLOW_CTX_DAG_ID=raizen_lab_01
AIRFLOW_CTX_TASK_ID=print_world
AIRFLOW_CTX_EXECUTION_DATE=2021-10-20T04:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-10-20T04:00:00+00:00
[2022-08-23 11:47:49,247] {logging_mixin.py:115} INFO - world
[2022-08-23 11:47:49,247] {python.py:173} INFO - Done. Returned value was: None
[2022-08-23 11:47:49,270] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=raizen_lab_01, task_id=print_world, execution_date=20211020T040000, start_date=20220823T114748, end_date=20220823T114749
[2022-08-23 11:47:49,326] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-23 11:47:49,377] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
